Header header

# references
geometry_msgs/Vector3 position_ref
geometry_msgs/Vector3 velocity_ref
geometry_msgs/Vector3 accel_ref
float64[3] yaw_ref
bool launch_flag
